actionpack/lib/action_dispatch/system_testing/browser.rb

```diff
@@ -1,14 +1,17 @@
+require "action_dispatch/system_testing/driver"
+
 module ActionDispatch
   module SystemTesting
-    class Browser # :nodoc:
+    class Browser < Driver # :nodoc:
       def initialize(name, screen_size)
+        super(name)
         @name = name
         @screen_size = screen_size
       end
 
-      def run
+      def use
```

Comment:
I change because the `Driver` and `Browser` classes don't run the driver, only sets the config - `Server` also doesn't `run` the puma server, only configures Capybara to use it. Was a bit confusing at first when reading through the code and I believe the public api could reflect more on what the class is doing.